style.css

```diff
@@ -0,0 +1,74 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5;
+  color: #212529;
+  text-align: left;
+  background-color: #fff; }
+
+.navbar-light{
+  background-color: ghostwhite ;
+}
+.navbar-brand {
+  color: #000000;
+  font-weight: 800;
+  font-size: 28px;
+  line-height: 1;
+ }
+.nav-link {
+  display: block;
+  padding: 0.5rem 1rem;
+  font-weight: 500;
+  position: relative;  
+ }
+
+ .nav-link:hover, .nav-link:focus {
+  text-decoration:none; 
+  font-weight: 700;
+  }
+
+
+  .nav-link.disabled {
+    color: #6c757d;
+    cursor: default; }
+
+
+.navbar-light .navbar-nav .nav-link:hover {
+color:green;
+}
+/* Top and Bottom borders go out */
+.topBotomBordersOut a:before, .topBotomBordersOut a:after
```

Comment:
Update this class such that only bottom animation comes in @media(max-width: 992px).